mplc/contributivity.py

```diff
@@ -938,6 +940,88 @@ def without_replacment_SMC(self, the_scenario, sv_accuracy=0.01, alpha=0.95):
             end = timer()
             self.computation_time_sec = end - start
 
+    # %% compute Partner value by reinforocment learning
```

Comment:
```suggestion
    # %% compute Partner value by reinforcement learning
```

Reply:
Is there a meaning for `%%`?

Reply:
I don't know 
Maybe in some IDE ? All the comments about the contributivity measures follow this format